packages/e2e/src/test-data/contacts-only-test-data.ts

```diff
@@ -0,0 +1,64 @@
+export const contacts: Array<{
+  firstName: string
+  lastName: string
+  primaryNumber: string
+  secondaryNumber: string
+  addressFirstLine: string
+  addressSecondLine: string
+  addToFavourites: boolean
+}> = [
```

Comment:
u can extract type of this Array to a type or interface :)

Reply:
https://github.com/mudita/mudita-center/pull/1201/commits/146739676ea20ca1dc855808c791cd6b3efc540c